pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -642,7 +648,7 @@ def _build_basis_and_op_matrices(self) -> None:
                 self.basis_name = "all"  # All three states
                 self.dim = 3
                 basis = ["r", "g", "h"]
-                projectors = ["gr", "hg", "rr", "gg", "hh"]
+                projectors = ["gr", "hg", "hr", "gg", "hh", "rr"]
```

Comment:
I thing you don't need this either

Reply:
Tests error without

Reply:
I'm curious what tests fail because I can't see where it is being used

Reply:
Quite a few tests fail. It happens when trying to call build_operator() without the operator in the op_matrix. Eg,
```
    def build_operator(self, operations: Union[list, tuple]) -> qutip.Qobj:
        """Creates an operator with non-trivial actions on some qubits.

        Takes as argument a list of tuples ``[(operator_1, qubits_1),
        (operator_2, qubits_2)...]``. Returns the operator given by the tensor
        product of {``operator_i`` applied on ``qubits_i``} and Id on the rest.
        ``(operator, 'global')`` returns the sum for all ``j`` of operator
        applied at ``qubit_j`` and identity elsewhere.

        Example for 4 qubits: ``[(Z, [1, 2]), (Y, [3])]`` returns `ZZYI`
        and ``[(X, 'global')]`` returns `XIII + IXII + IIXI + IIIX`

        Args:
            operations: List of tuples `(operator, qubits)`.
                `operator` can be a ``qutip.Quobj`` or a string key for
                ``self.op_matrix``. `qubits` is the list on which operator
                will be applied. The qubits can be passed as their
                index or their label in the register.

        Returns:
            The final operator.
        """
        op_list = [self.op_matrix["I"] for j in range(self._size)]

        if not isinstance(operations, list):
            operations = [operations]

        for operator, qubits in operations:
            if qubits == "global":
                return sum(
                    self.build_operator([(operator, [q_id])])
                    for q_id in self._qdict
                )
            else:
                qubits_set = set(qubits)
                if len(qubits_set) < len(qubits):
                    raise ValueError("Duplicate atom ids in argument list.")
                if not qubits_set.issubset(self._qdict.keys()):
                    raise ValueError(
                        "Invalid qubit names: "
                        f"{qubits_set - self._qdict.keys()}"
                    )
                if isinstance(operator, str):
                    try:
                        operator = self.op_matrix[operator]
                    except KeyError:
>                       raise ValueError(f"{operator} is not a valid operator")
E                       ValueError: sigma_rr is not a valid operator
```

Reply:
For sure we need `"rr"`, it was there before after all, I just don't see why we need `"hr"` (the one you added)